packages/devtools_app/lib/src/screens/memory/panes/chart/controller/chart_data.dart

```diff
@@ -11,13 +11,15 @@ import '../data/primitives.dart';
 
 // ignore: avoid_classes_with_only_static_members, enum-like classes are ok
 @visibleForTesting
-class Json {
-  static const isDeviceAndroid = 'isAndroid';
-  static const timeline = 'timeline';
-  static const interval = 'interval';
-  static const isLegendVisible = 'isLegendVisible';
+enum Json {
+  isDeviceAndroid('isAndroid'),
+  timeline('timeline'),
+  interval('interval'),
+  isLegendVisible('isLegendVisible');
```

Comment:
instead of passing the string representation to each of these, consider this pattern where the constructor has an optional name override, and then you don't have to pass in the string for the values where `Json.enumValue.name` is the equivalent. 

example: https://github.com/flutter/devtools/blob/master/packages/devtools_app/integration_test/test_infra/run/_test_app_driver.dart/#L417